src/components/pages/daysoff.style.js

```diff
@@ -1,7 +1,28 @@
 import { StyleSheet } from 'react-native';
 
-export default StyleSheet.create({
-    filterButton: {
-        fontSize: 20
-    }
-});
+export default (colors) => (
+    StyleSheet.create({
+        filterButton: {
+            fontSize: 20,
+            marginRight: 4
```

Comment:
Margin should be set on the indicator, as we don't want any margin when the indicator is not displayed